### DISCUSSION TITLE
New entries for the "Documentation" domain?

### DISCUSSION THREAD

**Discussion by bzg** [publiccodeyml/publiccode.yml#132] [Ideas]
Hi all, I would like to discuss possible new entries/keys for publiccode.yml.

If these suggestions are not relevant because the associated information for the proposed entries/keys is already supposed to be associated with existing ones, please let me know.

- "Documentation" is generic: what about distinguishing `User manual` and `Developer's documentation` (the existing `API Documentation` seems to call for the later)?

- On top of the `Homepage`, I'd suggest to have something like `Project documentation`, where users could reference a page giving general information on how the project is financed, led, etc.

Does any of this seem useful?

**Comment by bfabio** [comment]
Hi @bzg,

AFAIU
* [`description/[lang]/documentation`](https://yml.publiccode.tools/schema.core.html#key-description-lang-documentation) is the user manual
* [`description/[lang]/apiDocumentation`](https://yml.publiccode.tools/schema.core.html#key-description-lang-apidocumentation)  is the developers' documentation
* [`landingURL`](https://yml.publiccode.tools/schema.core.html#key-landingurl) is the generic project page, where you can put general information

Maybe we should clarify the Standard to better reflect that

**Comment by bzg** [comment]
Thanks for your answer!

Fabio Bonelli ***@***.***> writes:

>     description/[lang]/documentation is the user manual

"User manual" would sound more obvious.

>     description/[lang]/apiDocumentation is the developers' documentation

I'd favor having an "API reference" and a "developer's documentation", as they are distinct documents.  And many projects can have the latter while not having the former.

Is this something that has been raised before?

>     landingURL is the generic project page, where you can put general information

Yes, I get that.  My question was: would it be useful to have a new key for detailed information on how the project is funded, especially for public projects?  I'm not convinced myself this is needed, less keys is better.
